package.json

```diff
@@ -14,20 +14,28 @@
     "design system",
     "pattern library"
   ],
+  "workspaces": {
+    "nohoist": [],
+    "packages": [
+      "docs",
+      "packages/**"
+    ]
+  },
   "main": "packages/site/index.js",
   "scripts": {
-    "bootstrap": "lerna bootstrap --hoist && link-parent-bin",
-    "clean": "lerna clean",
+    "build": "lerna run build",
```

Comment:
i would like to use `wsrun` in the future but we have some cyclical dependencies we need to fix first